ui/src/app/workflows/components/workflow-dag/workflow-dag.tsx

```diff
@@ -210,103 +221,77 @@ export class WorkflowDag extends React.Component<WorkflowDagProps, WorkflowDagRe
                 }
                 consideredChildren.add(item.nodeName);
 
-                const node: NodeStatus = this.props.nodes[item.nodeName];
+                const node = props.nodes[item.nodeName];
```

Comment:
```suggestion
                const node = getNode(item.nodeName);
```

for consistency with the existing code, if we're already modifying it